src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/CacheConfig.kt

```diff
@@ -51,6 +51,18 @@ class CacheConfig {
             return ConcurrentMapCache(name, concurrentMap, true)
         }
     }
+
+    @Bean("historiskPensjonCache")
+    fun historiskPensjonkCache(): CacheManager = object : ConcurrentMapCacheManager() {
```

Comment:
Er det tilstrekkelig å kalle denne longCache? 
Dette er selve cachemanager, som sen oppretter en cache for `@Cacheable`